modules/olmv1-version-range-comparisons.adoc

```diff
@@ -0,0 +1,134 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="olmv1-version-range-comparisons_{context}"]
+= Version comparison strings
+
+You define a version range by adding a comparison string to the `spec.version` field in an Operator or extension's custom resource (CR). A comparison string is a list of space- or comma-separated values and one or more comparison operators. You can add an another comparison string by including an `OR`, or double vertical bar (`||`) comparison operator between the strings.
```

Comment:
Thinking also about "You define" at the beginning, but then "You can add" later. Not sure why, but the lack of "can" usually makes me feel funny, so maybe "You can define"?